index.html

```diff
@@ -5,12 +5,138 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose?styles=44178" rel="stylesheet">    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
   <link rel="stylesheet" href="style.css">  
   <title>World Developers Summit 2022</title>
 </head>
 <body>
-  <h1 id="h1">Hello World</h1> 
+  <header>
+    <div id="heading">
+        <div class="headingImg">
+            <img src="img/logos/facebook.png" class='fb' alt="">
+            <img src="img/logos/twitter.png" class="twt" alt="">
```

Comment:
Kindly add a meaningful `alt-text` to these `img` tags to follow the best practices 🤗